src/status_im/contexts/chat/messenger/messages/pin/events.cljs

```diff
@@ -127,5 +127,8 @@
 
 (rf/defn show-pins-bottom-sheet
   {:events [:pin-message/show-pins-bottom-sheet]}
-  [cofx chat-id]
-  (navigation/show-bottom-sheet cofx {:content (fn [] [pinned-messages-menu/view {:chat-id chat-id}])}))
+  [cofx chat-id & {:keys [inside-chat?] :or {inside-chat? true}}]
+  (navigation/show-bottom-sheet cofx
+                                {:content (fn [] [pinned-messages-menu/view
+                                                  {:chat-id      chat-id
+                                                   :inside-chat? inside-chat?}])}))
```

Comment:
Hi @codemaster115, Thank you for the PR and for fixing the issue. 

Should we instead use something like `:disable-message-long-press? (not= :chat (get-in cofx [:db :view-id]))`?

- inside-chat? prop is not self-explanatory and not clear why we are passing this prop.
- As long-press will be enabled by default, we don't need to worry about adding this prop to the messages list file or having default true.

Reply:
If for some reason, we need to use a message component someplace (apart from chat, maybe activity center, etc.) with long press functionality then we will be passing `:inside-chat?` true, which will be misleading.

Reply:
@Parveshdhull Yeah, it makes sense and I agree to your comment. 👍